srcpkgs/python3-shodan/template

```diff
@@ -13,7 +13,7 @@ maintainer="Alex Childs <misuchiru03+void@gmail.com>"
 license="MIT"
 homepage="https://developer.shodan.io"
 distfiles="https://github.com/achillean/shodan-python/archive/${version}.tar.gz"
-checksum=52932c6d508622877d0667e0290fee84e6cb749b6565b48ba359fbbf002f0845
+checksum=fc5f159f3edb6db0c2fff338b62f8a6c6eb63b12983bf729a4121fa30019753b
 make_check=extended
```

Comment:
that should be =no I think

Reply:
On the `make_check`? Because GitHub highlighted the checksum line(16) for me.

Reply:
It probably expects the comment next to $make_check or above it. @ericonr do you think `no` or `extended` makes more sense here? afaik we usually use extended for tests that take too long, not for tests that can't run without manually modifying the template to include a key.